scripts/fetch_target_logs.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+set -x
+
+KUBE_CONFIG="$(sudo find /tmp/ -type f -name "kubeconfig*")"
```

Comment:
we should export this from golang directly instead of using `cp`

Reply:
We need to keep it locally as , project-infra log collection will call log collection in-case of failure.

Reply:
We need to save it locally, so that both from project-infra and log collection and log collection from e2e can work with same file.

Reply:
But same kubeconfig that is saved in temp are used by project-infra script.